x-pack/plugins/security_solution/common/types/timeline/data_provider/index.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/** Represents the Timeline data providers */
+
+/** The `is` operator in a KQL query */
+export const IS_OPERATOR = ':';
```

Comment:
consider moving this to the `kbn-securitysolution-t-grid` package, because it overlaps with `x-pack/plugins/timelines/common/types/timeline/data_provider/index.ts`

Reply:
@kqualters-elastic will take care of it! ❤️